draft-packages/button/KaizenDraft/Button/styles.scss

```diff
@@ -28,6 +28,14 @@ $caButton-verticalPaddingForm: calc(
   transition: none; // override Murmur global styles :(
 }
 
+@mixin working-state {
+  pointer-events: none;
+
+  &:hover {
+    cursor: default;
+  }
```

Comment:
Looks like you don't actually need this, as long as you have the `pointer-events: none` above.

Reply:
This might actually go for all of the hover overrides that you have added below. I tried getting rid of them and didn't see any issues on hover.